test/framework/src/main/java/org/elasticsearch/search/aggregations/AggregatorTestCase.java

```diff
@@ -1487,12 +1502,22 @@ public AggTestConfig(IndexSearcher searcher, Query query, AggregationBuilder bui
             this(searcher, query, builder, DEFAULT_MAX_BUCKETS, randomBoolean(), true, fieldTypes);
         }
 
+        public AggTestConfig(
+            IndexSearcher searcher,
+            Query query,
+            AggregationBuilder builder,
+            boolean splitLeavesIntoSeparateAggregators,
+            MappedFieldType... fieldTypes
+        ) {
+            this(searcher, query, builder, DEFAULT_MAX_BUCKETS, splitLeavesIntoSeparateAggregators, true, fieldTypes);
+        }
```

Comment:
I think the extra constructor is too much.  There's already a `with...` method, just chain that onto the constructor calls.  I'm trying to avoid having a constructor for every set of parameters here.